helix-control-m/2-external-monitoring-tools-examples/alerts-to-helix-itsm/README.md

```diff
@@ -86,3 +87,20 @@ BMC Helix Remedy ITSM or on-prem with REST APIs enabled.
   * will compose a URL for the case to display, if the alert is that of a job.
   * There is a feature for those that want to send an email.
     * If you want to send an email and not the ticket, you will need to comment all lines that send the ticket and ensure variables set at the return are not used in messages later.
+
+## How to use it
+
+### Installation
+
+* Clone this repository
+
+* Install the Automation API CLI in the system you will use to receive the alerts and process them to forward to Helix ITSM.
+
+### Configuration
+
+* Modify your tktvars.json with the appropriate information for your environment
+  * You will need credentials and other information from both Helix Control-M and Helix ITSM
```

Comment:
What other information?

Reply:
The user will need to understand the code, adapt it to their needs and maintain it going forward. This is not intended to be a manual, but a simple readme page with basic information. I added what the users will need to be familiar with in the pre-requistes section

Reply:
The we need to link to the appropriate documentation.